microsite/src/main/tut/typeclass/Unfoldable1.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: docs
+title:  "Unfoldable1"
+---
+
+# Unfoldable1 [![GitHub](../img/github.png)](https://github.com/scalaz/scalaz/blob/series/8.0.x/base/shared/src/main/scala/scalaz/tc/unfoldable1.scala)
+
+* `Unfoldable1` identifies non-empty data structures which can be unfolded.*
+
+The generating function `f` in `unfoldRight1(f)(b)` always return a value, and then optionally a value to continue unfolding from.
+
+**Typical imports**
+
+```tut:silent
+import scalaz.tc._
+import scalaz.data._
+import scalaz.Scalaz._
+```
+
+# Usage
+
+TBD A sample with NonEmptyList
```

Comment:
If you expand this, I'll approve

Reply:
I pushed what I'd in mind, and tried a better wording as well.

Reply:
/cc @emilypi :)